Argo/Extensions/RawRepresentable.swift

```diff
@@ -1,3 +1,7 @@
+/**
+  Default implementation of `Decodable` for `RawRepresentable` types using
+  `String` as the raw value.
```

Comment:
I kinda think this might be better for full documentation, as opposed to inline documentation. I do think we could add more documentation around this though.